pkgs/misc/vscode-extensions/vscode-lldb/lldb/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, llvmPackages_8, cmake, python3, swig, libedit, breakpointHook }:
+
+let
+  version = "2018-12-06";
+  rev = "7f0cd38dae5cd77ac58adf97896ac0f41af759fc";
+  sha256 = "1z45wz0cfghirp2pwj6q8yqzpaafx5m7h206aj1wnqqb1k7qvwx4";
+
+  inherit (llvmPackages_8) llvm clang-unwrapped;
```

Comment:
why not `llvmPackages_latest`?

Reply:
The forked lldb should be compiled with the same version llvm. There are lots of API change between different llvm version.
It's hard to patch the source for latest llvm. (Need to dump all commits between official lldb_8..lldb_latest)

Reply:
I added a comment about this.

https://github.com/NixOS/nixpkgs/blob/083c45e5c465b21d168eb58679c4f0bcc7cd7879/pkgs/misc/vscode-extensions/vscode-lldb/lldb/default.nix#L8-L10

Reply:
Okay. If this builds and works (i.e. debugs well the Rust code) then let's get it merged soon. Then we can iterate.
I was able to swap in an `lldb_10` from the Rust repo (with modifications), so this shouldn't be too hard.

Reply:
@ggreif It is refactored and now uses https://github.com/vadimcn/llvm-project/tree/codelldb/11.0.0 which is based on lldb_11